src/libs/E2E/tests/linkingTest.e2e.ts

```diff
@@ -61,16 +59,7 @@ const test = (config: NativeConfig) => {
                 return;
             }
 
-            if (entry.name === CONST.TIMING.REPORT_INITIAL_RENDER) {
-                console.debug('[E2E] Navigating to linked report action…');
-                Timing.start(CONST.TIMING.SWITCH_REPORT);
-                Performance.markStart(CONST.TIMING.SWITCH_REPORT);
```

Comment:
@kirillzyusko I see we alter the way the app works here, shouldn't I bring back the markStart call for `open_report` here? It won't get fired otherwise because we're not pressing on the LHN.